crates/biome_cli/tests/snapshots/main_commands_lint/lint_help.snap

```diff
@@ -78,12 +78,14 @@ Available positional items:
 
 Available options:
         --write               Writes safe fixes
-        --suppress            Fix diagnostics with suppression comments if the language supports it.
         --unsafe              Allow to do unsafe fixes, should be used with `--write` or `--fix`
         --fix                 Alias for `--write`, writes safe fixes
         --apply               Alias for `--write`, writes safe fixes (deprecated, use `--write`)
         --apply-unsafe        Alias for `--write --unsafe`, writes safe and unsafe fixes
                               (deprecated, use `--write --unsafe`)
+        --suppress            Bulk fix diagnostics with suppression comments if the language
```

Comment:
I felt like it helped to add "Bulk" here to portray that you'd want to use this for migration purposes. Can take it off if desired.